recorder_package_name: arm_dmp_data
recorder_data_directory_name: dmp_data/demonstrations

create_directories: false

write_out_resampled_data: true
write_out_raw_data: false
write_out_clmc_data: false
write_out_statistics: false
